### ISSUE TITLE
Chatting and Error logging is not working for external users.

### ISSUE THREAD

**Issue by CuriousDrive** [CuriousDrive/BlazingChat#16]
there is a bug in signalr
the chat wont work if you register from facebook 
sugnalr want the clame id that facebook gave 
and not the id from localdb
i think you need another property that will store the clame id from external login
--> NameIdentifier <--
tnx for the video

**Comment by FrancoisBruere** [comment]
Yes. I managed to fix the issue. Thank you. By the way do you know where I can find solid information on email confirmation link to confirm email when users register / signup. I have tried a few solutions but non seem to be right. There doesn't seem to be any good documentation regarding this topic if you want to implement it from scratch.

Regards Francois
